gui/src/apis/mm-api.ts

```diff
@@ -68,18 +68,27 @@ class MessageMonitorApi {
     vehicleId,
     startTime,
     endTime,
+    long,
```

Comment:
Is this just a circular distance-based query? Why is it not sufficient to just query by incident ID?

Reply:
By incident ID I mean intersection ID lol. If there isn't a link to an intersection, then this looks great

Reply:
Yeah I was talking to John about using the originIp field but apparently one of the deployments has issues with the IP's being off for the intersections. So I decided to just do an offset lat/long based query to only get bsms in the general area of the intersection.

Reply:
makes sense to me